### PR TITLE
Added jsweek4 homework files.

### PR DESCRIPTION
**Pull request by sandhana14**

### CODE REVIEW

File: javascript/javascript1/week4/index.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Week4 Javascript</title>
+    </head>
+    <body>
+<script src = "jsweek4.js"></script>
+    </body>
+</html>
```
Comment (ajgcph2019) [inline javascript/javascript1/week4/index.html:9]: Good job. I have given feedback as below

File: javascript/javascript1/week4/jsweek4.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+function removeElement(command){
+    const removeElementFromTODo=command.substring(6,command.length-13);
+    for(let i=0;i<todo.length;i++){
+if(todo[i] === removeElementFromTODo){
+return todo.splice(i,1);
+}
+}
+}
+}
+
```
Comment (ajgcph2019) [inline javascript/javascript1/week4/jsweek4.js:114]: Do you use Prettier for formatting ? The function code block and the parenthesis dont seem formatted

File: javascript/javascript1/week4/jsweek4.js [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    function setTimer(){
+        console.log("Timer done");
+    }
+    console.log("Timer set for " + commandArray[4] +" "+ commandArray[5] );
+    setTimeout(setTimer, milliseconds);
+    return "Timer is set...now wait";
+    break;
+case "How many days until Christmas 2020":
+    var todaysDate = new Date();
+    var nextChris = new Date(2020, 11, 25);
```
Comment (ajgcph2019) [inline javascript/javascript1/week4/jsweek4.js:98]: Try to use const or let instead of var

File: javascript/javascript1/week4/jsweek4.js [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+    return "Please enter your command as a string which means in quotes";
+} 
+const commandArray = command.split(" "); 
+const pushArrActivity =command.substring(3,command.length-11);
+
+if(command){
+    const a = commandArray[2];
+    const b = commandArray[4];
+    let c;
+    if(commandArray[3] === "+"){
```
Comment (ajgcph2019) [inline javascript/javascript1/week4/jsweek4.js:16]: Use meaningful variable names

**Review by ajgcph2019 — CHANGES_REQUESTED** [review]

File: javascript/javascript1/week4/jsweek4.js [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+        c = Number(a)/Number(b);
+        return "Division of two numbers is " + c;
+    }else if(commandArray[3] === "*"){
+        c = Number(a)*Number(b);
+        return "Multiplication of two numbers is " + c;
+    }
+}
+switch(command){
+case "Hello my name is Sandhana":
+    if(name == ""){
```
Comment (ajgcph2019) [inline javascript/javascript1/week4/jsweek4.js:32]: What you did is correct. There is also an alternative like below 
There are string methods like 'startsWith' and 'endsWith'.
For example 
 const commandWords = inputString.split(' ');

  if (inputString.startsWith('hello my name is')) {
    name = commandWords[commandWords.length - 1];
    return `Nice to meet you ${name}`;
  }

File: javascript/javascript1/week4/jsweek4.js (hunk 2 above)
Comment (sandhana14) [reply javascript/javascript1/week4/jsweek4.js:114]: Hi Anna, No, I haven't used it. Hereafter, I will use it

File: javascript/javascript1/week4/jsweek4.js (hunk 5 above)
Comment (sandhana14) [reply javascript/javascript1/week4/jsweek4.js:32]: ok, Thank you for letting me know

**Pull request merged**